src/main/resources/db/migration/0250/V261__WithdrawalUndoDates.sql

```diff
@@ -0,0 +1,48 @@
+DROP VIEW nursery.withdrawal_summaries;
+
+-- Add undoes_withdrawal_date and undone_by_withdrawal_date
```

Comment:
I am wondering if we should include the old migration file name for views, so it is easier to find the original version for diffs (this one was easy to find!)

Reply:
Can't hurt!